src/java/main/org/apache/zookeeper/Login.java

```diff
@@ -197,10 +206,7 @@ public void run() {
                         break;
                     }
                     if (isUsingTicketCache) {
-                        String cmd = "/usr/bin/kinit";
-                        if (System.getProperty("zookeeper.kinit") != null) {
-                            cmd = System.getProperty("zookeeper.kinit");
-                        }
+                        String cmd = zkConfig.getProperty(ZKConfig.KINIT_COMMAND, KINIT_COMMAND_DEFAULT);
```

Comment:
It is relevant, but we have not done for other similar properties so leaving as it is